packages/gestalt/src/Video/Controls.js

```diff
@@ -151,7 +164,11 @@ function VideoControls({
       <Box padding={2}>
         <TapArea onTap={handlePlayingChange} fullWidth={false}>
           <Icon
-            accessibilityLabel={playing ? accessibilityPauseLabel : accessibilityPlayLabel}
+            accessibilityLabel={
```

Comment:
All labels should come from provider only

Reply:
DONE